apps/Pika Backup/install

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+flatpak_install org.gnome.World.PikaBackup || exit 1
```

Comment:
I just tried installing this on my 320bit Pi OS and it worked. However, most other flatpak apps don't work anymore because flatpak is pulling support for armhf. Any theories as to why this one still works?

Reply:
Some Flatpaks still have 32-bit support, not sure why..

Reply:
It could be that just outdated flatpaks still have armhf builds hanging around.
This flatpak app definitely seems outdated. When installing, I got these warnings:
```
Info: org.gnome.Platform//3.36 is end-of-life, with reason:
   The GNOME 3.36 runtime is no longer supported as of February 13, 2021. Please ask your application developer to migrate to a supported platform.
Info: org.freedesktop.Platform.GL.default//19.08 is end-of-life, with reason:
   The Freedesktop SDK 19.08 runtime is no longer supported as of September 1, 2021. Please ask your application developer to migrate to a supported version
```

Reply:
I believe that flatpak doesn't allow you to publish any new versions of a currently existing flatpak with armhf. so they are basically forcing devs to remove armhf from their published versions in order to update their apps

@Botspot which version of pika backup installed on armhf? if its 0.4.0 then that disproves my theory

Reply:
actually this is why:
```
armv7 support beyond 2021
Discussion if armv7 is needed after 2021 (ie, if it can be remove from the 20.08 runtime release)
Main concerns are maintenance costs by the number of users and hardware support to existing devices using flatpak (like endless ones)
Flatpak maintainers and Endless representatives agreed support until 2021 in the 19.08 fdsdk runtime should be enough
```

there are multiple rutime SDKs for flatpaks
Pika Backup uses the `org.gnome.Sdk` which DOES have armhf support and it has not been removed.
the `Freedesktop-sdk` is what has removed armhf in versions later than 19.0.8 as seen above

Reply:
> I believe that flatpak doesn't allow you to publish any new versions of a currently existing flatpak with armhf. so they are basically forcing devs to remove armhf from their published versions in order to update their apps
> 
> @Botspot which version of pika backup installed on armhf? if its 0.4.0 then that disproves my theory

I can't seem to find a version number for Pika itself, but the gnome platform version is 3.36, as stated above.
Here was the full output:
```
$ flatpak install flathub org.gnome.World.PikaBackup
Looking for matches…
Required runtime for org.gnome.World.PikaBackup/arm/stable (runtime/org.gnome.Platform/arm/3.36) found in remote flathub
Do you want to install it? [Y/n]: 
Info: org.gnome.Platform//3.36 is end-of-life, with reason:
   The GNOME 3.36 runtime is no longer supported as of February 13, 2021. Please ask your application developer to migrate to a supported platform.
Info: org.freedesktop.Platform.GL.default//19.08 is end-of-life, with reason:
   The Freedesktop SDK 19.08 runtime is no longer supported as of September 1, 2021. Please ask your application developer to migrate to a supported version

org.gnome.World.PikaBackup permissions:
    ipc                    network                       fallback-x11       ssh-auth       wayland       x11       devices       file access [1]
    dbus access [2]        system dbus access [3]

    [1] host, ~/.var/app/
    [2] org.freedesktop.FileManager1, org.freedesktop.Flatpak.*, org.freedesktop.secrets, org.gtk.vfs.*
    [3] org.freedesktop.login1


        ID                                   Branch          Op          Remote           Download
 1. [✓] org.gnome.Platform.Locale            3.36            i           flathub           17.8 kB / 323.2 MB
 2. [✓] org.gnome.Platform                   3.36            i           flathub          200.7 MB / 293.1 MB
 3. [✓] org.gnome.World.PikaBackup           stable          i           flathub            5.7 MB / 5.0 MB
```

Reply:
> I can't seem to find a version number for Pika itself, but the gnome platform version is 3.36, as stated above. Here was the full output:

ah if gnome version being installed is 0.36, then pika is old. the latest 0.4.0 version uses gnome 42 as seen here https://github.com/flathub/org.gnome.World.PikaBackup/blob/384be40b899de8a77b8828279284e17e673addea/org.gnome.World.PikaBackup.yml#L4

@Botspot if you run pika backup from the apps list, you can use t he hamburger menu to open the about page and get the version to verify

Reply:
> @Botspot if you run pika backup from the apps list, you can use the hamburger menu to open the about page and get the version to verify

It says 0.1.3

Reply:
> > @Botspot if you run pika backup from the apps list, you can use the hamburger menu to open the about page and get the version to verify
> 
> It says 0.1.3

ah so it is old and can not and will not be updated (I already checked on supported platforms like arm64 and x86_64 it is 1.4.0). I feel like we should not have an install-32 for this reason